src/stream/src/executor/mview/materialize.rs

```diff
@@ -1302,7 +1262,7 @@ mod tests {
                     .unwrap();
                 assert_eq!(
                     row,
-                    Some(OwnedRow::new(vec![Some(8_i32.into()), Some(3_i32.into())]))
+                    Some(OwnedRow::new(vec![Some(8_i32.into()), Some(2_i32.into())]))
```

Comment:
```SQL
U- 8 1
U+ 8 2
+  8 3
```
The conflict behavior is ignore. Quesiton is if we need to consider the update as a whole operation. The `U- 8 1` is trying to delete a non-existent record, so it is ignored. But if the following `U+ 8 2` should be ignored too?